servers/register_server_types.cpp

```diff
@@ -144,6 +145,9 @@ void register_server_types() {
 	ClassDB::register_virtual_class<PhysicsShapeQueryResult>();
 
 	ScriptDebuggerRemote::resource_usage_func = _debugger_get_resource_usage;
+
+	PhysicsServerManager::register_class(PhysicsServerSW::get_class_static());
+	PhysicsServerManager::set_default_class(PhysicsServerSW::get_class_static());
```

Comment:
Again, just use a function pointer that creates a PhysicsServer, this will allow it to work in GDNative too, which has no access to ClassDB.